pkg/component/extensions/extension/extension_test.go

```diff
@@ -165,6 +164,8 @@ var _ = Describe("Extension", func() {
 			},
 		}
 
+		fakeSeedClient = fakeclient.NewClientBuilder().WithScheme(kubernetes.SeedScheme).WithStatusSubresource(beforeExtension).Build()
```

Comment:
```suggestion
		fakeSeedClient = fakeclient.NewClientBuilder().WithScheme(kubernetes.SeedScheme).WithStatusSubresource(&extensionsv1alpha1.Extension{}).Build()
```
Similarly, for others.